templates/vue/src/components/tyde/activities/CircleOfSupport/onboarding/onboardingMachine.js

```diff
@@ -2,9 +2,16 @@ import { createMachine } from "xstate"
 
 export const OnboardingStates = {
   Welcome: "Welcome",
+  MoveConnections: "MoveConnections",
```

Comment:
I think it would look more clear if we order these in the way in which they appear